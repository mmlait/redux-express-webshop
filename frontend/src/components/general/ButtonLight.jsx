import React from 'react';
import styled from 'styled-components';
import colors from '../../colors';

const ButtonWrapper = styled.button`
  background-color: ${colors.primaryLight};
  color: ${colors.fontDark};
  padding: 15px;
  min-width: 100px;
  border: none;
  border-radius: 10px;
  cursor: pointer;
  &:hover {
    background-color: ${colors.btnLightHover};
  }
  &:focus {
    outline: none;
  }
`

const ButtonLight = ({onclick, type, content}) => {
  return (
    <ButtonWrapper onClick={onclick} type={type}>
      {content}
    </ButtonWrapper>
  )
}

export default ButtonLight;