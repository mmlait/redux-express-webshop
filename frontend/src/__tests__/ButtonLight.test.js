import React from 'react';
import ButtonLight from '../components/general/ButtonLight';
import { create } from 'react-test-renderer'

describe('ButtonLight component', () => {
    it('should render', () => {
    let tree = create(<ButtonLight />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})
