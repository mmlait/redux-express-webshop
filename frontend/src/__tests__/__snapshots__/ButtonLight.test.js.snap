// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLight component should render 1`] = `
<button
  className="sc-bdVaJa ecCyz"
/>
`;
